alerts/openshift-virtualization-operator/NodeStatusMaxImagesExceeded.md

```diff
@@ -0,0 +1,33 @@
+# NodeStatusMaxImagesExceeded
+
+## Meaning
+
+This alert fires when a node exceeds the configured maximum number of images
+reported in the node status (default: 50). Once this limit is reached, the
+scheduler might not be able to accurately schedule new pods because the reported
+image count is wrong.
+
+## Impact
+
+Pod scheduling might be imbalanced across nodes.
+
+## Diagnosis
+
+1. Obtain the image count for the node:
+
+   ```bash
+   $ oc get nodes <node> -o jsonpath='{range .status.images[*]}{.names}{"\n"}{end}' | wc -l
+   ```
+
+2. If image count is equal to the default (50) or your custom configured
```

Comment:
"If image count" > "If the image count"

"or your custom configured value" > "or to a configured value" 
- "custom" and "configured" are redundant, so you don't need both.

"resolve the issue" > "you must resolve the issue"
- sounds better

Reply:
updated